docs/hugo/content/_index.md

```diff
@@ -144,63 +189,47 @@ C:\> SET AZURE_CLIENT_SECRET=<your-client-secret>
 {{% /tab %}}
 {{< /tabpane >}}
 
-3. Install [the latest **v2+** Helm chart](https://github.com/Azure/azure-service-operator/tree/main/v2/charts):
+Then create a secret named `aso-credential` in the namespace you'd like to create ASO resources in.
 
 {{< tabpane text=true left=true >}}
 {{% tab header="**Shell**:" disabled=true /%}}
 {{% tab header="bash" %}}
-``` bash
-$ helm repo add aso2 https://raw.githubusercontent.com/Azure/azure-service-operator/main/v2/charts
-$ helm upgrade --install aso2 aso2/azure-service-operator \
-    --create-namespace \
-    --namespace=azureserviceoperator-system \
-    --set azureSubscriptionID=$AZURE_SUBSCRIPTION_ID \
-    --set azureTenantID=$AZURE_TENANT_ID \
-    --set azureClientID=$AZURE_CLIENT_ID \
-    --set azureClientSecret=$AZURE_CLIENT_SECRET \
-    --set crdPattern='resources.azure.com/*;containerservice.azure.com/*;keyvault.azure.com/*;managedidentity.azure.com/*;eventhub.azure.com/*'
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+ name: aso-credential
+ namespace: default
+stringData:
+ AZURE_SUBSCRIPTION_ID: "$AZURE_SUBSCRIPTION_ID"
+ AZURE_TENANT_ID: "$AZURE_TENANT_ID"
+ AZURE_CLIENT_ID: "$AZURE_CLIENT_ID"
+ AZURE_CLIENT_SECRET: "$AZURE_CLIENT_SECRET"
+EOF
 ```
-Note: **bash** requires the value for `crdPattern` to be quoted with `'` to avoid expansion of the wildcards.
 {{% /tab %}}
+
 {{% tab header="PowerShell" %}}
 ``` powershell
-PS> helm repo add aso2 https://raw.githubusercontent.com/Azure/azure-service-operator/main/v2/charts
-PS> helm upgrade --install aso2 aso2/azure-service-operator `
-    --create-namespace `
-    --namespace=azureserviceoperator-system `
-    --set azureSubscriptionID=$AZURE_SUBSCRIPTION_ID `
-    --set azureTenantID=$AZURE_TENANT_ID `
-    --set azureClientID=$AZURE_CLIENT_ID `
-    --set azureClientSecret=$AZURE_CLIENT_SECRET `
-    --set crdPattern=resources.azure.com/*;containerservice.azure.com/*;keyvault.azure.com/*;managedidentity.azure.com/*;eventhub.azure.com/*
-```
-{{% /tab %}}
-{{% tab header="CMD" %}}
```

Comment:
Why remove CMD? We know we have users using this.

Reply:
The problem is the file needs the env variables embedded in it... maybe I can write a multiline cmd that creates the file using the env variables... though not 100% sure how to do that without `EOF`

Reply:
Show a sample with placeholders and give them an instruction to replace the values?